chat_threads/threads/dao.py

```
@@ -0,0 +1,189 @@
+    def __init__(self, session: AsyncSession):
+        super().__init__(session=session, db_model=Thread)
+
+    async def get_thread_by_id(self, thread_id: UUID):
+        query = select(Thread).where(Thread.uuid == thread_id)
+        result = await self._execute_query(query)
+        thread = result.scalars().first()
+        return ThreadSchema.model_validate(thread)
+
+    async def update_thread(self, thread_uuid: UUID, update_values_dict: dict):
+        update_query = update(Thread).where(Thread.uuid == thread_uuid,
+                                            Thread.is_deleted==False).values(**update_values_dict)
```

Ruff (0.8.2)

Avoid equality comparisons to `False`; use `if not Thread.is_deleted:` for false checks

Replace with `not Thread.is_deleted`

(E712)